domaintools/api.py

```diff
@@ -354,17 +354,52 @@ def iris_investigate(self, domains=None, data_updated_after=None, expiration_dat
 
     def iris_detect_monitors(self, include_counts=False, datetime_counts_since=None, sort=None, order="desc", offset=0,
                              limit=None, **kwargs):
-        """Returns back a list of monitors based on the provided filters.
+        """Returns back a list of monitors in Iris Detect based on the provided filters.
         """
 
         if include_counts:
             if not datetime_counts_since:
                 raise ValueError('Need to define datetime_counts_since when include_counts is True')
             if isinstance(datetime_counts_since, datetime):
                 datetime_counts_since = str(datetime_counts_since.astimezone())
+            elif isinstance(datetime_counts_since, str):
+                kwargs["datetime_counts_since"] = datetime_counts_since
```

Comment:
As I was doing the New Domains endpoint it became clear we probably want to give the users an option to pass the datetime as a str or datetime object